src/libraries/native-binplace.proj

```diff
@@ -24,6 +24,7 @@
       <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
       <BinPlaceItem Include="$(NativeBinDir)*.dex" />
       <BinPlaceItem Include="$(NativeBinDir)*.jar" />
+      <BinPlaceItem Include="$(NativeBinDir)*.dat" />
```

Comment:
is this dealing also with ICU shards ?

Reply:
For browser we have this https://github.com/dotnet/runtime/pull/80421 
and for WASI we still need more https://github.com/dotnet/runtime/issues/82567

Reply:
It copies all the icu*.dat files just like mono.proj did.

Reply:
On second though, it possibly makes `ICULibNativeFiles` and `ICULibFiles` copying in wasm.proj and wasi.proj redundant.